docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/deployment.md

```diff
@@ -26,15 +26,31 @@ Follow this guide to deploy and run the service.
          ```bash
          make run ds-onvif-camera
          ```
+
+        ### Token Generation
         !!! note
             Need to wait for sometime for the services to be fully up before executing the next set of commands.
             Securely store Consul ACL token and the JWT token generated which are needed to map credentials and execute apis.
-            It is not recommended to store these secrets in cleartext in your machine.
+            It is not recommended to store these secrets in cleartext in your machine. 
 
+        !!! note
+            The JWT token expires after 119 minutes, and you will need to generate a new one.
+        
+        Generate the Consul ACL Token. Use the token generated anywhere you see `<consul-token>` in the documentation.
         ```bash
         make get-consul-acl-token
+        ```
+        Example output:
+        ```bash
+        12345678-abcd-1234-abcd-123456789abc
+        ```
+
+        Generate the JWT Token. Use the token generated anywhere you see `<jwt-token>` in the documentations
+        ```bash
         make get-token
         ```
+        Example output:
+        `eyJhbGciOiJFUzM4NCIsImtpZCI6IjUyNzM1NWU4LTQ0OWYtNDhhZC05ZGIwLTM4NTJjOTYxMjA4ZiJ9.eyJhdWQiOiJlZGdleCIsImV4cCI6MTY4NDk2MDI0MSwiaWF0IjoxNjg0OTU2NjQxLCJpc3MiOiIvdjEvaWRlbnRpdHkvb2lkYyIsIm5hbWUiOiJlZGdleHVzZXIiLCJuYW1lc3BhY2UiOiJyb290Iiwic3ViIjoiMGRjNThlNDMtNzBlNS1kMzRjLWIxM2QtZTkxNDM2ODQ5NWU0In0.oa8Fac9aXPptVmHVZ2vjymG4pIvF9R9PIzHrT3dAU11fepRi_rm7tSeq_VvBUOFDT_JHwxDngK1VqBVLRoYWtGSA2ewFtFjEJRj-l83Vz33KySy0rHteJIgVFVi1V7q5`
```

Comment:
This sample jwt token also should under bash as other examples are.

Reply:
I left it like that so that it would wrap the text. Otherwise it was all in one line that you had to scroll left and right to see